mantidproject/mantid
# D33 separate panels, simplify water normalisation

**Description of work.**
This PR targets 2 main features:

- For multipanel instruments as D33, it offers the possibility to generate outputs for each panel individually.
- Simplifies the normalisation to water, by removing the specific process type Reference; i.e. the water will act just as another sample.



**To test:**
Provided you have standard and system test data in your data search directories, set the default facility to ILL, and instrument to D33, run this:
```
config.appendDataSearchSubDir('ILL/D33/')
absorber = '002227'
tr_beam = '002192'
can_tr = '002193'
empty_beam = '002219'
can = '002228'
mask = 'D33Mask2.nxs'
SANSILLAutoProcess(
    SampleRuns='001464',
    SampleTransmissionRuns='002197',
    MaskFiles=mask,
    AbsorberRuns=absorber,
    BeamRuns=empty_beam,
    ContainerRuns=can,
    ContainerTransmissionRuns=can_tr,
    TransmissionBeamRuns=tr_beam,
    OutputWorkspace='iq',
    PanelOutputWorkspaces='panels'
)
```
Overplot the output in `iq` and the 5 workspaces within `panels`.
They should match up nicely.



Fixes #27705 . 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
